### ISSUE TITLE
Is there any plans to rewrite there examples with GTK4?

### ISSUE THREAD

**Issue by wszqkzqk** [gerito1/vala-gtk-examples#5]

**Comment by gerito1** [comment]
Yes. Plans to do it. But I didn't have the time or drive to do it yet.

I accept pull requests.

**Comment by chauhankiran** [comment]
Is is fine to update the example one at a time or all update in one PR?

**Comment by wszqkzqk** [comment]
> Is is fine to update the example one at a time or all update in one PR?

Maybe it can be separated into two directories and keep the GTK3 version until it is EOL？

**Comment by chauhankiran** [comment]
In that case, @gerito1 Can you please (re-)arrange the space for GTk4 examples to come?

**Comment by chauhankiran** [comment]
Quick question - These GTK3 example must be same and port to GTK4? Or we can do some modification? For example, I would like to simplify the `label.vala` to have only code related to `Label` only.

**Comment by gerito1** [comment]
@chauhankiran The examples can be whatever seems the simplest example, or more relevant one if the implementation call certain pattern.

About the arrangement for the GTK4 vs GTK3 it is not decided yet, but a directory is the preferred one.

@wszqkzqk EOL is on the future. When vala compiler decide that for us. I'm not concerned yet.
